analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/InvalidCastToInterface.cs

```diff
@@ -70,8 +70,8 @@ public class NullableTest
     public void Test1()
     {
         int? i1 = null;
-        var ii = (int)i1; // FIXME Non-compliant {{Nullable is known to be empty, this cast throws an exception.}}, this part is not based on SE
-//               ~~~~~~~
+        var ii = (int)i1; // Noncompliant {{Nullable is known to be empty, this cast throws an exception.}}, this part is not based on SE
```

Comment:
`this part is not based on SE`

Not sure I understand this part of the comment as the issue raised comes from the SE version of the rule.

Reply:
Not relevant anymore, SE part is gone. I've removed the annotations